Python 3.5 tests failing on async for ... in as_completed

Test failures like the following are now occuring on travis-ci.  This appears to be independent of any code change on our side.  I suspect that a depdendency has change.

```
__________________________ test_asyncio_as_completed ___________________________
    @coro_test
    async def test_asyncio_as_completed():
        async with AioClient(processes=False) as c:
            futures = c.map(inc, range(10))
    
            results = []
>           async for future in as_completed(futures):
E           TypeError: 'async for' received an invalid object from __aiter__: as_completed
```

@pitrou do you know of anything that might have caused this?

--- a/.travis.yml
+++ b/.travis.yml
@@ -7,10 +7,10 @@ services:
 env:
   matrix:
     - PYTHON=2.7 PACKAGES="python-blosc futures faulthandler"
-    - PYTHON=3.5 COVERAGE=true PACKAGES=python-blosc CRICK=true
+    - PYTHON=3.5.4 COVERAGE=true PACKAGES=python-blosc CRICK=true
     - PYTHON=3.6
     - HDFS=true PYTHON=2.7
-    - HDFS=true PYTHON=3.5 PACKAGES=python-blosc
+    - HDFS=true PYTHON=3.5.4 PACKAGES=python-blosc
 
 matrix:
   fast_finish: true

--- a/continuous_integration/travis/install.sh
+++ b/continuous_integration/travis/install.sh
@@ -49,6 +49,7 @@ conda install -q -c conda-forge \
     pytest=3.1 \
     pytest-faulthandler \
     pytest-timeout \
+    python=$PYTHON \
     requests \
     tblib \
     toolz \

